components/radio/skin.css

```diff
@@ -71,6 +71,15 @@ governing permissions and limitations under the License.
       color: var(--spectrum-radio-emphasized-text-color-down);
     }
   }
+  
+  & .spectrum-Radio-input:focus-ring,
+  &:hover .spectrum-Radio-input:focus-ring {
+    &:checked + .spectrum-Radio-button {
+      &:before {
+        border-color: var(--spectrum-radio-emphasized-circle-border-color-selected-key-focus);
+      }
+    }
+  }
```

Comment:
I also saw that there were slightly darker border-color vars defined for radios when they are selected and focused, updated accordingly. Lemme know if this doesn't match design and I'll update accordingly